ci/README.md

```diff
@@ -0,0 +1,23 @@
+## CI configurations and examples
+This folder holds vendors CIes configurations and examples. Configurations are used to control the vendors CIes behaviors, and examples are used as a reference for other vendors to be able to setup a CI of their own.
```

Comment:
first time seeing "CIes", can you just use CI throughout this file (or CIs)?

Reply:
done, changed it to `Vendors CI`